index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./styles.css">
+  <title>Portfolio</title>
+
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@300;400;500;600;700;800&family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
+</head>
+<body>
+  
+<header>
+  <nav>
+    <a href="#" class="logo">John Doe</a>
+    <div id="menu-button"><img id="humberger" src="./Images/Menu humberger icon.svg"></div>
+  </nav>
+</header>
+
+<section id="first">
+  <div id="fs-content">
+    <h1 id="greeting">Hey there. I'm Alex</h1>
+    <h3>I am a Software Developer</h3>
+    <p>
+      I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+      don't hesitate to contact me.
+    </p>
+    <ul id="soc-med">
+      <li>
+        <a>
+          <img src="./images/Github.svg" alt="GitHub">
+        </a>
+      </li>
+      <li>
+        <a>
+          <img src="./images/LinkedIn.svg" alt="LinkedIn">
+        </a>
+      </li>
+      <li>
+        <a>
+          <img src="./images/Peace-sign.svg" alt="Peace-sign">
+        </a>
+      </li>
+      <li>
+        <a>
+          <img src="./images/Twitter.svg" alt="Twitter">
+        </a>
+      </li>
+      <li>
+        <a>
+          <img src="./images/M.svg" alt="M">
```

Comment:
**[OPTIONAL]** - Please fix the src paths for your images.  Your image folder starts with a capital `I`.